Parkinson/Resources/JSONs/APHTasksAndSchedules.json

```diff
@@ -81,7 +69,7 @@
 		    "tasks"           : [
                 {
                     "taskTitle" : "PD Questionnaire",
-                    "taskID": "6-PDQ8-20EF83D2-E461-4C20-9024-F43FCAAAF4C8",
+                    "taskID": "PDQ8",
```

Comment:
looks like all the survey task IDs changed? was this to match what you were seeing coming in from the server? if so, cool